delete_book.php

```diff
@@ -0,0 +1,15 @@
+<?php
+include 'database.php';
+
+if (isset($_GET['id'])) {
```

Comment:
Consider refactoring the database operations into a repository class for better separation of concerns and maintainability.

```php
class BookRepository {
    private $connection;

    public function __construct($connection) {
        $this->connection = $connection;
    }

    public function deleteBook($id) {
        $stmt = $this->connection->prepare("DELETE FROM books WHERE id = ?");
        $stmt->bind_param("i", $id);
        return $stmt->execute();
    }
}
```

Usage:

```php
$bookRepository = new BookRepository($connection);
if ($bookRepository->deleteBook($id)) {
    echo "Book deleted successfully!";
    header("Location: list_books.php");
} else {
    echo "Error: " . $connection->error;
}
```